src/OpenTK/Platform/X11/X11GLNative.cs

```diff
@@ -476,9 +460,6 @@ private void SetWindowMinMax(short min_width, short min_height, short max_width,
 
             if (hints.flags != IntPtr.Zero)
             {
-                // The Metacity team has decided that they won't care about this when clicking the maximize
-                // icon, will maximize the window to fill the screen/parent no matter what.
-                // http://bugzilla.ximian.com/show_bug.cgi?id=80021
```

Comment:
Why remove this comment?

Reply:
The bug in question is (was) from c. 2002
It's no longer relevant to the current version of Metacity, and is useless fluff.

Reply:
Fair enough I guess